src/main/java/com/example/demo/service/JedisAutoServiceImpl.java

```diff
@@ -0,0 +1,44 @@
+package com.example.demo.service;
+
+import com.example.demo.dto.AutoDTO;
+import com.example.demo.entity.AutoModel;
+import com.example.demo.repositories.JedisAutoRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Optional;
+
+public class JedisAutoServiceImpl implements AutoService {
+
+    @Autowired
+    private JedisAutoRepository repository;
+
+    @Override
+    public AutoDTO create(AutoModel auto) {
+        repository.save(auto);
+        return AutoDTO.createAutoDTO(select(auto.getId()));
+    }
+
+    @Override
+    public AutoDTO update(AutoModel auto) {
+        return create(auto);
+    }
+
+    @Override
+    public void remove(long id) {
+        repository.deleteById(Long.toString(id));
+    }
+
+    @Override
+    public AutoModel select(long id) {
```

Comment:
should be `public AutoDTO select(long id) {`
and better to rename on `get` instead of `select`